README.md

```diff
@@ -185,3 +185,39 @@ memory prevents subsequent suites from running.
     `mbed_app.json`, and power cycle the target before and after
     the run to clear the memory. The total number of failures should match
     `CRYPTO.log` in [`test/logs`](./test/logs)`/<your-target>`.
+
+## Troubleshooting
+
+### Protected Storage (PS) test failures on Musca S1
+
+Since TF-M v1.3.0, Musca S1's Protected Storage (PS) is located in the target's QSPI flash.
+Normally tests should run without issues, but if PS test cases fail on Musca S1, you should
```

Comment:
Before TF-M v1.3, QSPI flash wasn't used anyway so switching from an older TF-M version may not be the problem. I think this is more useful if the user used QSPI for other purposes before, or the data is corrupted due to test being interrupted, or a certain commit on the TF-M master branch is temporarily problematic.

I don't know what happened in my case, but the erase fixed the problem. This info here is more for completeness, because putting the board in a clean state before running tests is a common thing to do and even TF-M's documentation recommends erasing PS and ITS (though it doesn't currently document how to do it for Musca S1's QSPI at all).

Reply:
Similarly we erase ITS too, but that's on MRAM so our application image already covers it. Though in most cases ITS erase isn't be needed either when tests do work properly.